src/access/AccessRestrictionProvider.cpp

```diff
@@ -197,13 +197,19 @@ CHIP_ERROR AccessRestrictionProvider::DoCheck(const std::vector<Entry> & entries
                 if (requestPath.requestType == RequestType::kAttributeReadRequest ||
                     requestPath.requestType == RequestType::kAttributeWriteRequest)
                 {
-                    return CHIP_ERROR_ACCESS_RESTRICTED_BY_ARL;
+                    if (!IsGlobalAttribute(requestPath.entityId.value()))
+                    {
+                        return CHIP_ERROR_ACCESS_RESTRICTED_BY_ARL;
+                    }
                 }
                 break;
             case Type::kAttributeWriteForbidden:
                 if (requestPath.requestType == RequestType::kAttributeWriteRequest)
                 {
-                    return CHIP_ERROR_ACCESS_RESTRICTED_BY_ARL;
+                    if (!IsGlobalAttribute(requestPath.entityId.value()))
```

Comment:
I don't believe any of the global attributes are writable, but perhaps one will be in the future

Reply:
+1. I think it's fine to leave that here, but yeah, not applicable really...